### DISCUSSION TITLE
Disclaimer in custom.js - Unterstrich in URL führt zu Ersetzung durch <em>

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#223] [Q&A]
Wenn in der custom.js im Disclaimer eine URL eingefügt wird, welche mehrere Unterstrich enthält, werden der erste und der letzte durch "em" ersetzt.
gdi_win_svc/temporaere_luftfahrthindernisse_bin/Anleitung_Erfassung-temp-LFH.pdf
wird zu
gdi "em" win_svc/temporaere_luftfahrthindernisse_bin/Anleitung "/em" Erfassung-temp-LFH.pdf
Die URL kann daher nicht aufgerufen werden.
Mir ist bislang nicht gelungen, das Problem zu beheben.

**Comment by gis-eni** [comment]
Kannst du bitte genauer beschreiben was du tust. Keine Ahnung wie du in der ``custom.js`` Disclaimer umsetzt hast.

**Reply by faulgurke** [reply]
Gerne.
In der custom.js füge ich ein "webgis.confirm" ein, welches in der "message" einen weiterführenden Link aufweist.
"[Anleitung:](https://.../gdi_win_svc/temporaere_luftfahrthindernisse_bin/Anleitung_Erfassung_temp_LFH.pdf)".
In obiger URL sind Unterstriche vorhanden. Beim Aufruf des Links bekomme ich eine "Not Found" Fehlermeldung, weil der erste (nach "gdi"und der letzte Unterstrich (nach "temp" durch "em" ersetzt worden sind. Der Unterstrich wird als Steuerzeichen für die Formatierung "italic" interpretiert.
So sieht das dann aus:
gdi "em" win_svc/temporaere_luftfahrthindernisse_bin/Anleitung_Erfassung_temp "/em"LFH.pdf. (Die spitzen Klammern vor und nach "em" habe ich durch Hochkommata ersetzt, weil auch hier der Text dann in "italic" dargestellt wird.
Die Frage ist: Wie kann ich verhindern, dass "_" durch "em" ersetzt wird und dann mein Link nicht mehr funktioniert.

**Reply by gis-eni** [reply]
bitte das code snippet dazu

**Reply by faulgurke** [reply]
webgis.confirm({
                 title: "Wichtige Hinweise zur Erfassung temporärer Luftfahrthindernisse",
                 height: "350px",
                 message:"md:Es wird darauf hingewiesen, dass es sich bei dem digitalen Meldetool für zeitweilige (temporäre) Luftfahrthindernisse um ein sicherheitsrelevantes Datenauslieferungsprogramm handelt." + "\n"
				 "[Anleitung](https://vogis.cnv.at/gdi_win_svc/temporaere_luftfahrthindernisse_bin/Anleitung_Erfassung_temp_LFH.pdf)" + "\n" 
                 ,
                 okText: "Gelesen und verstanden - **weiter zur Karte**",
                 cancelText: "Nein, danke!",
                 
                 //onOk: function() {
                 //	 webgis.localStorage.set('gelesen', '1');
                 //},
                 onCancel: function () {
                     document.location = "https://cnvbregdit1.net.vlr.gv.at/webgis5_portal/testportal"
                 }
            });
    });
};

**Reply by gis-eni** [reply]
Ok, das Problem liegt hier am Markdown konverter. Du hast den String mit "md:" markiert, damit er formiert wird. Einige Formatierungsregeln greifen jetzt leider auch innerhalb des Links, was zu dem oben beschriebenen Problem führt.

Des Problem sollte mit dem nächsten Release >= 6.24.4901 behoben sein.
